modules/02b-cdc-connect-debezium.md

```diff
@@ -0,0 +1,120 @@
+# Module 02 - CDC with Apache Kafka Connect and Debezium
+
+## Deploy and check PostgreSQL database
+
+Deploy the PostgreSQL database used as storage for devices information with some pre-crated data.
+
+```shell
+oc adm policy add-scc-to-user anyuid -z postgres
+oc apply -f kafka-connect-debezium/postgres/postgres.yaml
+```
+
+Check that the table `deviceinfo` is prepopulated with some devices related information.
+
+```shell
+oc exec $(oc get pods --selector=app=postgres -o=jsonpath='{.items[0].metadata.name}') -- env PGOPTIONS="--search_path=devices" psql -U postgres -c "SELECT * FROM deviceinfo;"
+```
+
+## Deploy Apache Kafka Connect
+
+The Apache Kafka Connect cluster can be deployed using the cluster operator.
+There are two available CRDs (Custom Resource Definitions) for that: `KafkaConnect` and `KafkaConnectS2I`.
+The `KafkaConnect` CRD uses Kafka Connect image as a base layer and the connector plugins are injected by building new image on top of it.
+The `KafkaConnectS2I` CRD leverages the Source-2-Image OpenShift's feature for handling builds and adding connectors plugin.
+In this workshop we will use `KafkaConnect` for adding the Debezium PostgreSQL plugin connector.
```

Comment:
Maybe we can make the other module-02 as an appendix and adding here a link saying "if you want to use KafkaConnect S2I ...."